ajax/libs/jeditable.js/package.json

```diff
@@ -1,17 +1,30 @@
 {
   "name": "jeditable.js",
-  "filename": "jeditable.min.js",
-  "version": "1.7.3",
-  "author": "Mika Tuupola",
-  "description": ". With few lines of JavaScript code I allow you to click and edit the content of different html elements. I am based on Dylan Verheul’s editable.",
+  "filename": "jquery.jeditable.min.js",
+  "version": "1.8.0",
+  "author": {
+    "name": "Mika Tuupola",
+    "email": "tuupola@appelsiini.net",
+    "web": "http://www.appelsiini.net/projects/jeditable"
+  },
+  "description": "jQuery edit in place plugin. Extendable via plugin architecture. Plugins for plugin. Really.",
   "homepage": "http://www.appelsiini.net/projects/jeditable",
   "keywords": [
     "editable",
     "jquery"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/tuupola/jquery_jeditable.git"
+    "url": "https://github.com/NicolasCARPi/jquery_jeditable.git"
   },
+  "npmName": "jquery-jeditable",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
```

Comment:
Let's also add the files which can be used in front-end under `src`.
The whole auto-update config is 
```js
  "npmName": "jquery-jeditable",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    },
    {
      "basePath": "src",
      "files": [
        "jquery.jeditable*"
      ]
    }
  ],
```

This will be a great work.
Thank you.